sonic-slave/Dockerfile

```diff
@@ -227,6 +227,9 @@ RUN apt-get update && apt-get install -y \
         python3-requests \
         python3-pytest \
         python3-colorama
```

Comment:
please add \

Reply:
yep, i just fixed it, this one was a last-minute issue while merging my patch with latest github code...